bin/load-dev-fixtures.php

```diff
@@ -15,17 +18,37 @@
 }
 $connection->executeQuery($query, array(), array());
 
-// Add some dummy threads
-$threadContent = <<<EOT
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut posuere sapien ut tellus cursus, ut auctor urna malesuada. Mauris pretium ut nisl at vulputate. Duis varius elementum tortor, rutrum porta velit vehicula eu. Sed porttitor molestie porttitor. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Integer et porttitor erat. Fusce pharetra arcu nec bibendum aliquam. Praesent eleifend ipsum lacinia, ornare quam id, dignissim turpis. Vivamus velit ligula, maximus placerat hendrerit quis, molestie ac felis. Ut ligula arcu, laoreet vel elementum ac, varius id ligula.
+// Add some dummy data
+$passwordEncryptionService = new PasswordEncryptionService();
```

Comment:
on line 8 we require services.php so all the services that get required here are available in this script. So you already have the password encryption service in the scope of this script. You just have to use it.